ntbs-service/Pages/Alerts/TransferRequest.cshtml

```diff
@@ -33,8 +33,12 @@
                               ref="errorField" asp-validation-for="TransferRequest.TbServiceCode"></span>
                         <autocomplete-select ref="filterContainer" v-on:mounted="filteringMounted" v-on:input-changed="filteringChanged"
                                              placeholder="Select TB service" emit-change-on="select" inline-template>
-                            <select ref="selectField" nhs-select-type="@tbServiceCodeSelectErrorState" asp-for="TransferRequest.TbServiceCode"
-                                asp-items="Model.TbServices">
+                            @* below id necessary to disable automcomplete in chrome - see https://adamsilver.io/blog/stopping-chrome-from-ignoring-autocomplete-off/ *@

```

Comment:
Surely we want to do this for all `autocomplete-select` components, for the same reason?

Reply:
The strange thing is I checked the others and they weren't having the same issue... But I suppose it's probably worth doing to protect against it in the future?

Reply:
I'm getting the same behaviour as you, but it does seem very odd. I think we should make the change just in case though.

Reply:
I've added id's to all autocomplete-dropdown components now!